src/throttle.cpp

```diff
@@ -53,7 +53,7 @@ uint16_t Throttle::ReadAcceleratorPress()
     uint16_t max_throttle_vol = 92; // usually switches between 91 and 92
     uint16_t tolerance = 10;
 
-    float throttle_pos;
+    uint16_t throttle_pos;
```

Comment:
could you do some math after this to allow some amount of buffer? (as we discussed in the PDR, the drivers tend to ride the brakes so we want the first x percentage of pedal press to equivalent to throttle pos 0, or you can check it later and set torque to 0 but i think changing the min position might make more sense)